src/components/common/Avatar/Avatar.js

```diff
@@ -0,0 +1,12 @@
+import styles from "./Avatar.module.css";
+
+ // Avatar Component
+ function Avatar() {
+  
+   return(
+     <img src="https://static.pexels.com/photos/324658/pexels-photo-324658.jpeg" alt="profile pic" className={styles.Avatar}/>
+   );
+ }
```

Comment:
let's use arrow function syntax for more compact view
```suggestion
 const Avatar = () => (
     <img src="https://static.pexels.com/photos/324658/pexels-photo-324658.jpeg" alt="profile pic" className={styles.Avatar}/>
   );
```